packages/orf/orf.1.0.1/opam

```diff
@@ -0,0 +1,64 @@
+opam-version: "2.0"
+authors: "Francois Berenger"
+maintainer: "unixjunkie@sdf.org"
+homepage: "https://github.com/UnixJunkie/orf"
+bug-reports: "https://github.com/UnixJunkie/orf/issues"
+dev-repo: "git+https://github.com/UnixJunkie/orf.git"
+license: "LGPL-2.1-or-later WITH OCaml-LGPL-linking-exception"
+build: ["dune" "build" "-p" name "-j" jobs]
+depends: [
+  "batteries" {>= "3.2.0"}
+  "cpm" {>= "11.0.0"}
+  "dolog" {>= "4.0.0"}
+  "dune" {>= "2.8"}
+  "minicli"
+  "molenc"
```

Comment:
Hi, @UnixJunkie! Just checking in here whether we can fix the lower bound here

Since the versions of dep used on ocaml 5 work, could we just constraint this to the earliest `molenc` that works on ocaml 5? Iiuc that would be

```suggestion
  "molenc" {>= "16.15.0"}
```

WDYT?